frontend/src/modules/scaffold/backboneFormsOverrides.js

```diff
@@ -90,7 +90,16 @@ define([
         disableNativeSpellChecker: false,
         enterMode: CKEDITOR[Origin.constants.ckEditorEnterMode],
         entities: false,
-        extraAllowedContent: Origin.constants.ckEditorExtraAllowedContent,
+        htmlSupport: {
+          allow: [
+            {
+              "name": new RegExp(Origin.constants.htmlSupportAllowed.name),
+              "attributes": Origin.constants.htmlSupportAllowed.attributes,
+              "classes": Origin.constants.htmlSupportAllowed.classes,
+              "styles": Origin.constants.htmlSupportAllowed.styles
```

Comment:
I don't understand where you're getting the sub properties of this object from `Origin.constants.htmlSupportAllowed.*` `name, attributes, classes, styles` when you've defined the value as a string `htmlSupportAllowed: '.*',`

![image](https://github.com/user-attachments/assets/285bf5a7-baae-4025-b88f-6112cc465a49)

Reply:
Sorry, overlooked this. This file is only used when installing a new AAT. I was using my own `conf/config.json` file, which doesn't get added to git.